packages/website/blog/2023-07-09-announcing-typescript-eslint-v6.md

```diff
@@ -619,7 +619,7 @@ For more information, see:
 
 ### Package Exports
 
-The v5 `@typescript-eslint/` packages don't use [Node.js package.json exports](https://nodejs.org/api/packages.html#package-entry-points), so anyone can import any file in any package by directly referencing a path within the dist folder.
+The v5 `@typescript-eslint/` packages don't use [Node.js package.json exports](https://nodejs.org/api/packages.html#package-entry-points), which allows someone to import any file in any package by directly referencing a path within the `dist` directory.
```

Comment:
🤔 I don't see any advantage to the new sentence form here. It's a bit longer and conveys the same information. Is there a reason you'd prefer the new form?

Reply:
1) the word "directory" is more agnostic and should be generally preferred over "folder", since "folder" is specifically a Windows word.
2) `dist` should probably be in code ticks to signify that it is a directory instead of a word.
3) "anyone --> any file --> any package" might be a bit confusing.
the important part is the "any file in any package" part i think.
we are not just talking about "anyone", we are specifically talking about "end-users who invoke the TSESLint API directly in their code", NOT people who just use the TSESLint plugin more conventionally.
maybe this can be workshopped further.

Reply:
Got it - you raise good points!
* I think the difference between "directory" and "folder" is pretty low priority for us, and "directory" is longer to read ever so slightly... but yeah makes sense
* +1 on the ticks
* Agreed that the original phrasing is confusing and also that I do want to simplify the new phrasing a bit...

Maybe:

```suggestion
The v5 `@typescript-eslint/*` packages don't use [Node.js package.json exports](https://nodejs.org/api/packages.html#package-entry-points), which allows importing any file in any package by directly referencing a path within the package's `dist/` directory.
```

How do you feel about that?

Reply:
lgtm